open-company/open-company-web
# Invite logo

Card: https://trello.com/c/bvDYPT6k

Review with
- auth https://github.com/open-company/open-company-auth/pull/34
- email https://github.com/open-company/open-company-email/pull/15

To test:
- make sure your org has a logo
- invite someone
- [x] did you receive the invitation email? Good
- [x] do you see the company logo at the top? Good
- now switch to an org w/o logo (or remove it from the current one)
- invite someone
- [x] did you receive the invitation email? Good
- [x] do you NOT see the company logo at the top? Good
